python/BristolNTuple_Electrons_cfi.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
 nTupleElectrons = cms.EDProducer("BristolNTuple_Electrons",
-    InputTag = cms.InputTag('slimmedElectrons'),
+    InputTag = cms.InputTag('calibratedPatElectrons'),
```

Comment:
to be precise, this value should always be `electronUserData`, see [python/run/miniAODToNTuple_cfg.py#L475](https://github.com/DouglasBurns/NTupleProduction/blob/e922415fce22c873e1ef13526fda2431684a12eb/python/run/miniAODToNTuple_cfg.py#L475)